zebra-network/src/config.rs

```diff
@@ -347,7 +347,6 @@ impl<'de> Deserialize<'de> for Config {
 
         let config = DConfig::deserialize(deserializer)?;
 
-        // TODO: perform listener DNS lookups asynchronously with a timeout (#1631)
```

Comment:
Do we instead want to update this comment to `WONTFIX` instead of `TODO`?

Reply:
It turns out we actually implemented this change, it's in `Config::resolve_peers()`. So there is nothing left to do (or to WONTFIX!).